patterns/direct_requestor_blocking.py

```diff
@@ -0,0 +1,121 @@
+    "solace.messaging.service.vpn-name": os.environ.get('SOLACE_VPN') or "default",
+    "solace.messaging.authentication.scheme.basic.username": os.environ.get('SOLACE_USERNAME') or "default",
+    "solace.messaging.authentication.scheme.basic.password": os.environ.get('SOLACE_PASSWORD') or ""
+    }
+
+# Build A messaging service with a reconnection strategy of 20 retries over an interval of 3 seconds
+# Note: The reconnections strategy could also be configured using the broker properties object
+messaging_service = MessagingService.builder().from_properties(broker_props)\
+                    .with_reconnection_retry_strategy(RetryStrategy.parametrized_retry(20,3))\
+                    .build()
+
+# Blocking connect thread
+messaging_service.connect()
+print(f'\nMessaging Service connected? {messaging_service.is_connected}')
+
+# Event Handling for the messaging service
+service_handler = ServiceEventHandler()
+messaging_service.add_reconnection_listener(service_handler)
+messaging_service.add_reconnection_attempt_listener(service_handler)
+messaging_service.add_service_interruption_listener(service_handler)
+
+# Create a direct message requestor and register the error handler
+direct_requestor_blocking = messaging_service.request_reply() \
+                                    .create_request_reply_message_publisher_builder() \
+                                    .build()
+
+# Blocking Start thread
+direct_requestor_blocking.start()
+print(f'\nDirect Requestor ready? {direct_requestor_blocking.is_ready()}')
+
+# Prepare outbound message payload and body
+message_body = "Hi, My name is " + f'{name}'
+outbound_msg_builder = messaging_service.message_builder() \
+                .with_property("application", "samples") \
+                .with_property("language", "Python")
+
+# Capture the timestamp and use that as message-id
+gmt = time.gmtime()
+message_id = calendar.timegm(gmt)
+
+print('\nSend a KeyboardInterrupt to stop publishing')
+try: 
+    print(f'============================')
+    topic = Topic.of(TOPIC_PREFIX + '/direct/hello/'  + f'{unique_name}')
+    print(f'Publishing to topic:\n{topic}')
+
+    try:
+        # Direct publish the message with dynamic headers and payload
+        outbound_msg = outbound_msg_builder \
+                            .with_application_message_id(f'NEW {message_id}')\
+                            .build(f'\n{message_body}')
+        print(f'>>>>>>>>>>>>>>>>>>>>>>>>>>>>')
+        print(f'Publishing request (body):' + outbound_msg.get_payload_as_string())
+        print(f'----------------------------')
+        print(f'Publishing message:\n{outbound_msg}')
+
+        response = direct_requestor_blocking.publish_await_response(request_message=outbound_msg, \
+                                                                    request_destination=topic, \
+                                                                    reply_timeout=10000)
```

Comment:
let's change the timeout to `3000` and follow the same convention in https://github.com/SolaceSamples/solace-samples-java-jcsmp/blob/master/src/main/java/com/solace/samples/jcsmp/patterns/DirectRequestorBlocking.java#L45

Reply:
Done